submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.ts

```diff
@@ -116,6 +127,31 @@ export class ExperimentInfoComponent implements OnInit {
     );
   }
 
+  durationHandle(secs: number) {
+    var hr = Math.floor(secs / 3600);
+    var min = Math.floor((secs - hr * 3600) / 60);
+    var sec = Math.round(secs) - hr * 3600 - min * 60;
+    var showHr;
+    var showMin;
+    var showSec;
```

Comment:
```suggestion
    const hr = Math.floor(secs / 3600);
    const min = Math.floor((secs - hr * 3600) / 60);
    const sec = Math.round(secs) - hr * 3600 - min * 60;
    let showHr;
    let showMin;
    let showSec;
```

Reply:
@pingsutw  Thanks your suggestion. I have updated this pr.